lesson_2/Lesson_2.2.ipynb

```diff
@@ -0,0 +1,204 @@
+{
```

Comment:
<p>in the modern <s>workl</s> <strong>world</strong></p><p>Nitpick: MCMC is not <em>that</em> novel</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Removed novel</p>